spearhead/assets/sass/_woocommerce.scss

```diff
@@ -0,0 +1,24 @@
+#content .wc-block-grid .wc-block-grid__product-onsale {
+	font-family: var(--global--font-secondary);
+}
+
+body[class*="woocommerce"] #page span.onsale,
+#content .wc-block-grid .wc-block-grid__product .wc-block-grid__product-onsale {
```

Comment:
I know this is not specific to this PR but something 'inherited' from Seedlet, but I wonder what's the rationale to add the `#content` ID selector here. In the future it might be possible to add these blocks in other areas like the footer, in that case these styles would not be applied, correct? Was that intentional?

Reply:
Thanks for pointing that out, it's worth looking into this to improve Seedlet for the future